libs/etterlatte-ktor/src/main/kotlin/TokenUtils.kt

```diff
@@ -35,7 +35,16 @@ inline val ApplicationCall.brukerTokenInfo: BrukerTokenInfo
                     val sub = it.getClaim(Claims.sub)
                     Pair(oid, sub)
                 }
-        val saksbehandler = claims?.getClaim(Claims.NAVident)
+        val saksbehandler =
+            claims?.getClaim(Claims.NAVident)
+                ?: (
+                    if (oidSub?.first?.equals(oidSub.second) == true) {
```

Comment:
Kan du her bruke `erSystembruker`-metoden frå BrukerTokenInfo? Eller motsatt? Trur det er lurt å ha éin plass vi avgjer om det er systembrukar eller ikkje.

Reply:
Ja, det skal la seg gjøre! Tror at hele "saksbehandler"-logikken i denne blokka her er unødvendig og kan overlates til `BrukerTokenInfo`